.github/workflows/beima.yml

```diff
@@ -1,9 +1,8 @@
 name: beima
 
 on:
-  pull_request:
```

Comment:
I would think that we would want to keep this no?

Reply:
I think once we get every command we need, we can change it back. I put on push in here, so whenever a push happens on any branch it will run. It may be useful as we are adding stuff to it and working out the kinks

Reply:
Having both the Push and PR trigger caused the workflow to run twice every time I would push to my branch that was in a PR. So I think for now the Push trigger will cover what we need. We can change it how to something a bit more specific once we get all the commands we need in the build and test scripts.